doc/getting_started/index.rst

```diff
@@ -3,319 +3,409 @@
+         ``.bashrc`` file ensures this setting is permanent.
+
+   .. group-tab:: macOS
+
+      #. On macOS, install :program:`Homebrew` by following instructions on the `Homebrew
+         site`_, and as shown here. Homebrew is a free and open-source package management system that
+         simplifies installing software on macOS.  While installing Homebrew,
+         you may be prompted to install additional missing dependencies; please follow
+         any such instructions as well.
+
+         .. code-block:: bash
+
+            /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+
+      #. Then, install these host dependencies with the ``brew`` command:
+
+         .. code-block:: bash
+
+            brew install cmake ninja gperf ccache dfu-util qemu dtc python3
+
+      #. Install west:
+
+         .. code-block:: bash
+
+            pip3 install west
 
-Follow an operating system specific guide, then come back to this page.
+      .. _Homebrew site: https://brew.sh/
 
-.. toctree::
-   :maxdepth: 1
+   .. group-tab:: Windows
 
-   Linux <installation_linux.rst>
-   macOS <installation_mac.rst>
-   Windows <installation_win.rst>
+      .. note:: Currently, the built-in `Windows Subsystem for Linux (WSL)
+         <https://msdn.microsoft.com/en-us/commandline/wsl/install_guide>`__
+         doesn't support flashing your application to the board.  As such,
+         we don't recommend using WSL yet.
+
+      These instructions assume you are using the Windows ``cmd.exe``
+      command prompt. Some of the details, such as setting environment
+      variables, may differ if you are using PowerShell.
+
+      An easy way to install native Windows dependencies is to first install
+      `Chocolatey`_, a package manager for Windows.  If you prefer to install
+      dependencies manually, you can also download the required programs from their
+      respective websites and verify they can be found on your PATH.
+
+      #. Install :program:`Chocolatey` by following the instructions on the
+         `Chocolatey install`_ page.
+
+      #. Open a command prompt (``cmd.exe``) as an **Administrator** (press the
+         Windows key, type "cmd.exe" in the prompt, then right-click the result and
+         choose "Run as Administrator").
+
+      #. Disable global confirmation to avoid having to confirm
+         installation of individual programs:
+
+         .. code-block:: console
+
+            choco feature enable -n allowGlobalConfirmation
+
+      #. Install CMake:
+
+         .. code-block:: console
+
+            choco install cmake --installargs 'ADD_CMAKE_TO_PATH=System'
+
+      #. Install the rest of the tools:
+
+         .. code-block:: console
+
+            choco install git python ninja dtc-msys2 gperf
+
+      #. Close the Administrator command prompt window.
+
+      #. Install west:
+
+         .. code-block:: bash
```

Comment:
Please use the 'bat' lexer (not bash) for windows cmd.exe syntax highlighting. Some of these lines look weird rendered for bash.